astronomer/providers/google/cloud/xcom_backends/gcs.py

```diff
@@ -0,0 +1,74 @@
+import json
+import os
+import pickle  # nosec
+import uuid
+from typing import Any
+
+import pandas as pd
+from airflow.configuration import conf
+from airflow.models.xcom import BaseXCom
+from airflow.providers.google.cloud.hooks.gcs import GCSHook
+
+
+class GCSXComBackend(BaseXCom):
+    """
+    Custom XCom persistence class extends base to support various datatypes.
+    To use this XCom Backend, add the environment variable `AIRFLOW__CORE__XCOM_BACKEND`
+    to your environment and set it to
+    `astronomer.providers.google.cloud.xcom_backends.gcs.GCSXComBackend`
+    """
+
+    PREFIX = os.getenv("PREFIX", "gcs_xcom_")
+    GCP_CONN_ID = os.getenv("CONNECTION_NAME", "google_cloud_default")
```

Comment:
`CONNECTION_NAME` is a very generic name for an env var

Reply:
Fixed